client/trino-cli/src/main/java/io/trino/cli/Console.java

```diff
@@ -83,6 +87,7 @@
         implements Callable<Integer>
 {
     public static final Set<String> STATEMENT_DELIMITERS = ImmutableSet.of(";", "\\G");
+    private static final SqlParser SQL_PARSER = new SqlParser();
```

Comment:
I’m not sure I like this. The Trino parser is not a library meant to be used outside of the server. We intentionally tried to keep the dependency between the CLI and the parser as limited as possible (currently, just the lexer). 

The dependency between the CLI and the parser is the only reason the parser module still has to work with Java 8, while the rest of the codebase has moved on. We should not make this even harder by further coupling the two.